db/migrate/20200828174620_add_shipping_option_filter_to_mirakl_store.rb

```diff
@@ -0,0 +1,5 @@
+class AddShippingOptionFilterToMiraklStore < ActiveRecord::Migration
+  def change
+    add_column :spree_mirakl_stores, :shipping_method_display_rate, :string, default: 'both'
```

Comment:
Shouldn't this be an int? Why is this text?

Reply:
This is how spree does it. For some reason, their dropdown maps to 
```DISPLAY = [:both, :front_end, :back_end]

    # Used for #refresh_rates
    DISPLAY_ON_FRONT_AND_BACK_END = 0
    DISPLAY_ON_FRONT_END = 1
    DISPLAY_ON_BACK_END = 2```
so we use display like they do and then map it to the const values